app-modules/engagement/tests/Feature/Actions/DeliverEngagementsTest.php

```diff
@@ -135,3 +140,20 @@
     // This engagement should not be picked up and delivered
     Queue::assertPushed(EngagementEmailChannelDelivery::class, 0);
 });
+
+it('will only dispatch a job to send an engagement that is scheduled', function () {
+    Queue::fake(EngagementEmailChannelDelivery::class);
+    Notification::fake();
+
+    // Given that we have an engagement that is not scheduled but should otherwise be delivered
+    Engagement::factory()
+        ->deliverNow()
+        ->has(EngagementDeliverable::factory()->email()->count(1))
+        ->create();
+
+    // When our job runs to pick up engagements
+    DeliverEngagements::dispatchSync();
+
+    // This engagement should not be picked up and delivered
+    Queue::assertPushed(EngagementEmailChannelDelivery::class, 0);
```

Comment:
Hmm, I don't know if I fully understand scheduled.

How does the engagement get sent if it is not scheduled?

Reply:
@Orrison There is currently a difference between how the Message Center and every other piece of the application handles engagements. The Message Center has a dedicated action to synchronously dispatch engagements (as there is no option to schedule), while every other engagement creation method has the option to schedule, and relies on the job in the kernel to dispatch it. 

The addition of this column is a way to differentiate items dispatched synchronously through the message center, and those created in other places. The idea is that this will eliminate the potential of an engagement being picked up by the ever running process after it has already been synchronously dispatched.